src/xrScriptEngine/script_engine.cpp

```diff
@@ -786,9 +797,8 @@ void CScriptEngine::setup_callbacks()
 #if !XRAY_EXCEPTIONS
         luabind::set_error_callback(CScriptEngine::lua_error);
 #endif
-#ifndef MASTER_GOLD
```

Comment:
This allows modmakers to see meaningful script error messages in the log. as @ShokerStlk explained.